addons/product/models/product_template.py

```diff
@@ -311,6 +311,11 @@ def _onchange_uom_id(self):
         if self.uom_id:
             self.uom_po_id = self.uom_id.id
 
+    @api.onchange('type')
+    def _onchange_type(self):
+        # Do nothing but needed for inheritance
+        return
```

Comment:
It seems to me a bit shitty to do that, do you confirm @sle-odoo ? But since I don't know what modules will be installed or not, I can't say what `_onchange_type` will be the first one except for this one here of course.
The best solution is maybe rename the *stock.product.template* `_onchange_type`  ?

Reply:
i gues it's the proper way but an onchange could return a direct, i would return {} here and properly return in the overrides